index.html

```diff
@@ -9,5 +9,16 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
+
+    <div class="title-container">
+        <img width="50px" height="50px" src="./assets/triangle.svg" alt="triangle">
```

Comment:
Always put styles in one `styles.scss`file. 
For example in this case i would create an image-container style with `width= ... and height= ...`